recipe/meta.yaml

```diff
@@ -19,7 +18,7 @@ requirements:
     - {{ compiler("cxx") }}
     - cmake
     - ninja
-    - python 2.7.*
+    - python
```

Comment:
Hopefully this fixes the package conflicts seen on [Windows CI builds]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=369778&view=logs&jobId=a70f640f-cc53-5cd3-6cdc-236a1aa90802&j=a70f640f-cc53-5cd3-6cdc-236a1aa90802&t=f5d15007-a01c-5ad8-c9ce-4d519d3b275f ). Though not sure if the build scripts were migrated to Python 3 yet